app/routes/preprints/provider/notifications.js

```diff
@@ -0,0 +1,15 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+
+export default Route.extend({
+    theme: service(),
+
+    model() {
+        const providerId = this.get('theme.id');
+        const subscriptionId = `${providerId}_new_pending_submissions`;
```

Comment:
Is there a benefit to assigning these in the model hook instead of in the controller?